oauth2-client/src/test/java/net/oauth2/client/AutoRenewingTokenProviderTest.java

```diff
@@ -88,7 +86,7 @@ public void testDelayDuration()  {
 		given(trs.tokenRenewTask.getToken()).willReturn(expectedTemporalToken);		
 		Duration delayDuration = trs.estimatedRepetitionsDelay();
 		assertNotNull(delayDuration);
-		assertTrue(delayDuration.compareTo(Duration.ofSeconds(1L)) == 0);
+		assertEquals(0, delayDuration.compareTo(Duration.ofSeconds(1L)));
```

Comment:
Where suggested by the IDE, simplified assertions.   Will get better statements about what the expected and actual values were this way.